Lombiq.Hosting.Tenants.EnvironmentRobots.Tests.UI/Extensions/TestCaseUITestContextExtensions.cs

```diff
@@ -0,0 +1,41 @@
+using Atata;
+using Lombiq.Tests.UI.Extensions;
+using Lombiq.Tests.UI.Services;
+using OpenQA.Selenium;
+using Shouldly;
+using System.Threading.Tasks;
+
+namespace Lombiq.Hosting.Tenants.EnvironmentRobots.Tests.UI.Extensions;
+
+public static class TestCaseUITestContextExtensions
+{
+    public static async Task TestRobotMetaTagIsMissingAsync(this UITestContext context, bool shouldBeMissing)
+    {
+        var metaTagXPath = By.XPath($"//meta[@name='robots' and @content='noindex, nofollow']").OfAnyVisibility();
+
+        await context.SignInDirectlyAsync();
+        await context.GoToHomePageAsync();
+
+        // Checking the response header with JavaScript.
```

Comment:
This is apparent from the code, but explain _why_ do we use JS.

Reply:
I think if sufficient now but as it is not my request please resolve it if it is good for you